crates/fluvio/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fluvio"
-version = "0.14.0"
```

Comment:
need to bump to `0.15.0` with API change

Reply:
done